### PR TITLE
Badge

### PR DESCRIPTION
**Pull request by sharathnair9999**
added badges on avatar and icon
badge-showcase
s

### CODE REVIEW

**Review by junaidshaikh-js — COMMENTED** [review]
Looks good to me.

File: pages/docs.html [hunk 1]
```diff
@@ -69,27 +71,37 @@ <h3>How to use?</h3>
...
           </code>
         </pre>
       </section>
-   
+
       <section class="code-container">
         <pre class="pre-code">
           <code class="language-css">
 @import url("https://nothing-ui-library.netlify.app/styles/index.css")
           </code>
```
Comment (VishalPatil18) [inline pages/docs.html:87]: Code indentation could be improved here. It looks a bit off.

File: pages/docs.html [hunk 2]
```diff
@@ -108,76 +120,73 @@ <h3 id="alert">Alert</h3>
         for each context such as primary, success, error and warning. To use these alerts, use the contextual classes
         (e.g., <code>.alert-success</code>).</p>
```
Comment (VishalPatil18) [inline pages/docs.html:121]: You could improve upon the code indentation here. 
e.g. 
```
<p class="section-text">
    This component is used to inform the user about any form of alerts that are distinguished by the class name
    for each context such as primary, success, error and warning.
    To use these alerts, use the contextual classes (e.g. <code>.alert-success</code>).
</p>
```

File: pages/docs.html [hunk 3]
```diff
@@ -108,76 +120,73 @@ <h3 id="alert">Alert</h3>
...
+    &lt;p class="alert-message">This is a warning alert. You get a close icon as well&lt;/p>
+    &lt;i class="far fa-times-circle alert-close fa-lg">&lt;/i>
   &lt;/div>
   &lt;br>
   &lt;div class="alert alert-simple">
-    &lt;p class="alert-message">This is a simple alert. You get a close icon as well&lt;/p>&lt;img
-      src="../styles/components/alert/close.svg" alt="close-icon" class="alert-close">
+    &lt;p class="alert-message">This is a simple alert. You get a close icon as well&lt;/p>
+    &lt;i class="far fa-times-circle alert-close fa-lg">&lt;/i>
   &lt;/div>
```
Comment (VishalPatil18) [inline pages/docs.html:181]: Here also the Code indentation looks broken.

File: styles/components/alert/alert.css [hunk 4]
```diff
@@ -72,10 +72,10 @@
   color: var(--primary-900);
   background-color: var(--primary-200);
 }
-
+/* 
 .alert-primary .alert-close{
   filter: invert(35%) sepia(27%) saturate(4472%) hue-rotate(177deg) brightness(96%) contrast(98%);
-}
+} */
```
Comment (VishalPatil18) [inline styles/components/alert/alert.css:78]: You could try avoid adding commented code in the PR. It's a bad practice as suggested by our MA Pranshu bhaiya.

File: styles/components/badges/badge.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+.icon-badge{
+  font-size: 0.6rem;
+  border-radius: 50%;
+  top: -0.6rem;
+  height: 1.3rem;
+  position: absolute;
+  right: -0.8rem;
+  font-weight: bolder;
+  width: 1.3rem;
+  border: 1px solid #727272;
```
Comment (VishalPatil18) [inline styles/components/badges/badge.css:89]: As there are multiple properties in this class, you could sort them in alphabetical order as that helps in debugging as well as code maintainability and readability as suggested by Vikram Sir.

**Review by VishalPatil18 — CHANGES_REQUESTED** [review]
- A few suggestions were added mostly for code indentation. 
- The main body has a horizontal scroll which you could fix. 

Apart from this the UI as well as the code looks good. Great work @sharathnair9999 👍

File: pages/docs.html (hunk 3 above)
Comment (sharathnair9999) [reply pages/docs.html:181]: the code snippet frame is also getting indented if i indent the html file. and as a result the in the output the code snippet is having empty space on the left which makes it look weird. tried carbon.now.sh but it ain't responsive

Comment (sharathnair9999) [reply pages/docs.html:181]: > Here also the Code indentation looks broken.
the code snippet frame is also getting indented if i indent the html file. and as a result the in the output the code snippet is having empty space on the left which makes it look weird. tried carbon.now.sh but it ain't responsive

File: styles/components/alert/alert.css (hunk 4 above)
Comment (sharathnair9999) [reply styles/components/alert/alert.css:78]: yeah removed it

File: styles/components/badges/badge.css (hunk 5 above)
Comment (sharathnair9999) [reply styles/components/badges/badge.css:89]: > As there are multiple properties in this class, you could sort them in alphabetical order as that helps in debugging as well as code maintainability and readability as suggested by Vikram Sir.

done. thanks for bringing this up

**Pull request merged**